config-model-generator/Makefile

```diff
@@ -7,7 +7,6 @@ docker_tag:
 docker_push:
 
 java_build:
-	./build-config-models.sh build
```

Comment:
How does this empty target work? Doesn't it exclude the Maven build from happening when `make all` is used?

Reply:
Ahhh, good point.